accounts/keystore/file_cache.go

```diff
@@ -65,7 +64,11 @@ func (fc *fileCache) scan(keyDir string) (mapset.Set, mapset.Set, mapset.Set, er
 		// Gather the set of all and fresly modified files
 		all.Add(path)
 
-		modified := fi.ModTime()
+		info, err := fi.Info()
+		if err != nil {
+			return nil, nil, nil, err
+		}
+		modified := info.ModTime()
```

Comment:
These changes require a bit more overview, and should perhaps not be mixed in with such a large PR. How about, 
- Make one PR which does only trivial changes (replace ioutil with io). 
- A different PR replacing ioutil with os

And also, there are a lot of whitespace-changes, I'm not sure why those came into it, the code _should_ already be formatted properly. Removing the whitespace-changes should also make the PR(s) smaller

Reply:
I totally understand. I've removed `ReadFile` and the whitespace.
- I'll create another PR to address `ReadFile`
- Sorry about the whitespace. Forgot to disable trim trailing whitespace in VSCode.